lib/error.js

```diff
@@ -0,0 +1,36 @@
+/** takes an error object and serializes it to a plan object */
+function serializedError (input) {
+  if (!(input instanceof Error)) {
+    const error = new Error('attempted to serialize a non-error object')
+    return serializedError(error)
+  }
+  return {
+    errorType: input?.constructor?.name,
```

Comment:
If we reach here, we know `input` is defined.

```suggestion
    errorType: input.constructor?.name,
```

Reply:
We aren't getting custom error types with `input.name` so `AxiosError` and the like would just show as `Error`.

```js
> class CustomError extends Error {}
undefined
> const e = new CustomError('xxx')
undefined
> e.name
'Error'
> e.constructor.name
'CustomError'
```